src/Output/QRPbm.php

```diff
@@ -0,0 +1,50 @@
+<?php
+/**
+ * Class QRPbm
+ *
+ * @created      11.12.2025
+ * @author       wgevaert
+ * @copyright    2025 wgevaert
+ * @license      MIT
+ */
+declare(strict_types=1);
+
+namespace chillerlan\QRCode\Output;
+
+use chillerlan\QRCode\Output\QROutputAbstract;
+
+class QRPbm extends QROutputAbstract {
+
+	private const LIGHT_DARK = [ '0', '1' ];
+
+	protected function prepareModuleValue( mixed $value ): mixed {
+		return $value;
+	}
+
+	protected function getDefaultModuleValue( bool $isDark ): mixed {
+		return $isDark ? self::LIGHT_DARK[1] : self::LIGHT_DARK[0];
```

Comment:
this can return `(int)$isDark` for the b/w PBM, 0 and 255 (or 65535?) for grayscale PGM, and for the RGB formats PPM/PAM it could return an array similar to the GD output class.